modules/nf-core/vsearch/sort/main.nf

```diff
@@ -0,0 +1,36 @@
+process VSEARCH_SORT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::vsearch=2.21.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vsearch:2.21.1--h95f258a_0':
+        'quay.io/biocontainers/vsearch:2.21.1--h95f258a_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val sort_arg
+
+    output:
+    tuple val(meta), path("*_sorted.fasta"), emit: fasta
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    vsearch \\
+        $sort_arg $fasta \\
+        --threads $task.cpus \\
+        --output ${prefix}_sorted.fasta \\
```

Comment:
```suggestion
        --output ${prefix}.fasta \\
```
We discussed this a while back and by supplying a set suffix for files we remove some flexibility for the users (instead we opted to add some sort of an error message to the modules, see: https://nf-co.re/docs/contributing/modules#naming-conventions part 6)

Reply:
I missed this one, thanks!

Reply:
No problem 😄